src/net.h

```diff
@@ -789,7 +792,7 @@ class CConnman
 
     CConnman(uint64_t seed0, uint64_t seed1, AddrMan& addrman, bool network_active = true);
     ~CConnman();
-    bool Start(CScheduler& scheduler, const Options& options);
+    bool Start(CScheduler& scheduler, const Options& options) EXCLUSIVE_LOCKS_REQUIRED(!m_total_bytes_sent_mutex);
```

Comment:
e59472f8af0a1962fa5c331e41fa22a4f8602db8 In the corresponding definition in `src/net.cpp`
```diff
 bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
 {
+    AssertLockNotHeld(m_total_bytes_sent_mutex);
     Init(connOptions);
```

Reply:
Done. Thanks.